CHANGELOG.md

```diff
@@ -1,3 +1,88 @@
+## [Unreleased][unreleased]
+
+### Changed
+
+##### Configuration
+
+- Drop the `lua_code_cache` configuration property. This setting has been
+  considered harmful since 0.11.0 as it interferes with Kong's internals.
+  [#2854](https://github.com/Mashape/kong/pull/2854)
+
+### Fixed
+
+##### Core
+
+- Fixed an edge-case where `preserve_host` would sometimes craft an upstream
+  request with a Host header from a previous client request instead of the
+  current one.
+  [#2832](https://github.com/Mashape/kong/pull/2832)
+- Ensure APIs with regex URIs are evaluated in the order that they are created.
+  [#2924](https://github.com/Mashape/kong/pull/2924)
+- Fixed a typo that prevented the load balancing components to respect
+  the Upstream `slots` property.
```

Comment:
Fixed a typo that prevented the load balancing components from respecting the Upstream `slots` property.
--or--
Fixed a typo that caused the load balancing components to ignore the Upstream `slots` property.

Reply:
Oh my - that read so bad... Thanks! 😅